README.md

```diff
@@ -86,15 +86,8 @@ jobs:
         with:
           platform: ${{ matrix.platform }}
           arch: ${{ matrix.arch }}
-          float-precision: single
-          build-target-type: template_release
-      - name: 🔗 GDExtension Build
```

Comment:
Pretty sure this preset was just broken, as `matrix.float-precision` wasn't defined.